src/AirBench/Api/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,192 @@
+﻿using AirBench.Api.Models;
+using AirBench.Api.Repositories;
+using AirBench.Models;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace AirBench.Api.Controllers
+{
+    public class ReviewController : ApiController
+    {
+        private IReviewApiRepository _repo;
+
+        public ReviewController(IReviewApiRepository repo)
+        {
+            _repo = repo;
+        }
+
+        /// <summary>
+        /// Attempts to add a review to the bench with the given ID.
+        /// 
+        /// ROUTE:
+        /// "review/add/{id}"
+        /// 
+        /// REQUEST BODY:
+        /// {
+        ///     "Description": `string`,
+        ///     "Rating": `int`
+        /// }
+        /// 
+        /// RESPONSE BODY:
+        /// {
+        ///     "Success": `bool`,
+        ///     "Id": `int`,
+        ///     "BenchId": `int`,
+        ///     "Description": `string`,
+        ///     "Rating": `int`
+        /// }
+        /// </summary>
+        /// <param name="id">The bench ID.</param>
+        /// <param name="request">The request body.</param>
+        /// <returns>The response body.</returns>
+        [HttpPost]
+        public async Task<ReviewAddResponse> Add(int id, ReviewAddRequest request)
+        {
+            var review = new Review()
+            {
+                BenchId = id,
+                Description = request.Description,
+                Rating = request.Rating
```

Comment:
Reviews/comments also require a field for DateTime or DateTimeOffset when the comment was created on, and something to link them to the author since you need to also list the author's name and last name initial at the comment. 

You'd need to likely possibly change that in the database and in the relevant data structures (Review.cs, ReviewAddRequest.cs, etc). Here you could just then do something simple like CreatedOn = DateTimeOffset.now; 

For the author I see that your customprincipal has an Id field and you can probably pass it here so you don't have to query the DB for it.